volumina/widgets/dataExportOptionsDlg.py

```diff
@@ -135,6 +137,12 @@ def __init__(self, parent, opDataExport):
         # See self.eventFilter()
         self.installEventFilter(self)
 
+        def pressedResetButton():
```

Comment:
```suggestion
        def resetButtonClicked():
```

if we _have_ to do some bikeshedding I'd prefer to name it the other way round and also use the `clicked` signal which seems to be more standard, rather than the `pressed` signal.

Reply:
Absolutely fine by me :)